src/test/java/seedu/sudohr/logic/commands/department/RemoveEmployeeFromDepartmentCommandTest.java

```diff
@@ -0,0 +1,291 @@
+package seedu.sudohr.logic.commands.department;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static seedu.sudohr.testutil.Assert.assertThrows;
+
+import java.nio.file.Path;
+import java.util.function.Predicate;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.sudohr.commons.core.GuiSettings;
+import seedu.sudohr.logic.commands.CommandResult;
+import seedu.sudohr.logic.commands.exceptions.CommandException;
+import seedu.sudohr.model.Model;
+import seedu.sudohr.model.ReadOnlySudoHr;
+import seedu.sudohr.model.ReadOnlyUserPrefs;
+import seedu.sudohr.model.SudoHr;
+import seedu.sudohr.model.department.Department;
+import seedu.sudohr.model.department.DepartmentName;
+import seedu.sudohr.model.employee.Employee;
+import seedu.sudohr.model.employee.Id;
+import seedu.sudohr.testutil.TypicalDepartmentNames;
+import seedu.sudohr.testutil.TypicalEmployees;
+
+public class RemoveEmployeeFromDepartmentCommandTest {
+
+    // Handle removing an employee
+    @Test
+    public void execute_employeeAcceptedByDepartment_addSuccessful() throws CommandException {
+
+        RemoveEmployeeFromDepartmentCommandTest.ModelStubAcceptingEmployeeRemoved modelStub =
+                new RemoveEmployeeFromDepartmentCommandTest.ModelStubAcceptingEmployeeRemoved();
+        modelStub.addEmployee(TypicalEmployees.ALICE);
+        Department toAdd = new Department(TypicalDepartmentNames.DEPARTMENT_NAME_FIRST);
+        modelStub.addDepartment(toAdd);
+
+        new AddEmployeeToDepartmentCommand(new Id("101"),
```

Comment:
rather than relying on this class, possible to directly add employee to department using any of the avaiable methods? Was thinking if we wanna keep the test function as 'pure' as possible.

Reply:
true, i'll look into it